core/tee/tee_svc_storage.c

```diff
@@ -104,29 +103,6 @@ TEE_Result tee_svc_storage_create_filename(void *buf, size_t blen,
 	return TEE_SUCCESS;
 }
 
-#ifdef CFG_REE_FS
-/* "/dirf.db" or "/<file number>" */
-TEE_Result
-tee_svc_storage_create_filename_dfh(void *buf, size_t blen,
-				    const struct tee_fs_dirfile_fileh *dfh)
-{
-	char *file = buf;
-	size_t pos = 0;
-	size_t l;
-
-	if (pos >= blen)
-		return TEE_ERROR_SHORT_BUFFER;
-
-	file[pos] = '/';
-	pos++;
-	if (pos >= blen)
-		return TEE_ERROR_SHORT_BUFFER;
-
-	l = blen - pos;
-	return tee_fs_dirfile_fileh_to_fname(dfh, file + pos, &l);
-}
-#endif
-
 /* "/TA_uuid" */
 TEE_Result tee_svc_storage_create_dirname(void *buf, size_t blen,
```

Comment:
How about moving this function and `tee_svc_storage_create_filename()` to `tee_rpmb_fs.c` following the same logic?

Reply:
Yes, will do.